### ISSUE TITLE
Field loses focus after keystroke

### ISSUE THREAD

**Issue by dmvtech** [microsoft/BotFramework-Composer#8576]
## Describe the bug

When clicking into the `Allow Interruptions` field and then typing a character, the focus is lost after the first key stroke. I do not know all fields this is limited to. In _my_ case, I am hitting the 'delete' key on my keyboard. But I am hitting the delete key multiple times (as i'm intending to delete multiple characters). This is a disastrous situation in my case as the focus **then** goes to the central designer pane and then the delete key deletes my **prompt** that I have created and configured already.

## Version

Version:  2.1.0-nightly.264266.2a99641
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## To Reproduce

1. Create an expression in the Allow interruptions field
2. Click away (say to center/designer pane) and then back to "save" (this does not reproduce if you have **_just_** entered in the expression.
3. Click into the field to edit
4. Hit any keystroke
5. Notice focus is lost

## Expected behavior

Field does not lose cursor/keyboard focus.

## Screenshots

![losingfocus](https://user-images.githubusercontent.com/43397658/129952362-6959d7a4-f8c9-425e-8629-5f3aa8a65298.gif)

## Additional context

It's not just the 'delete' key, but any key. As far as I can tell. Here is a screenshot of hitting the 't' key. The `t` character is entered, focus is lost (and the suggestion/intelisense is shown)
![image](https://user-images.githubusercontent.com/43397658/129952528-af9d119a-d62d-41b2-8ec8-75dd913ee8a6.png)

**Comment by dmvtech** [comment]
Possibly related?: https://github.com/microsoft/BotFramework-Composer/issues/8491 (keystrokes and focus)

**Comment by tonyanziano** [comment]
Can you see if this reproduces in the web build?

**Comment by dmvtech** [comment]
Yes @tonyanziano It reproduces with my build of `2.1.0` and `Microsoft Edge
Version 92.0.902.73 (Official build) (64-bit)`.

**Comment by tonyanziano** [comment]
Seems like this is not an Electron issue then, and could be a React re-rendering issue.
